suite/tests/common/getretaddr.c

```diff
@@ -112,6 +112,9 @@ GLOBAL_LABEL(FUNCNAME:)
         mov      x0, x30
         ldr      x30, [sp], #16
         ret
+# elif defined(RISCV64)
+        /*# FIXME i#3544: Port tests to RISC-V64*/
```

Comment:
Please use standard syntax: no #; end punctuation and a space before the closing *.  Ditto below.